### ISSUE TITLE
Map not shown on image information view

### ISSUE THREAD

**Issue by mzur** [biigle/geo#59]
The geo map is not shown on the image information view (`/images/:id`). This should be fixed.

**Comment by mtiessen1175** [comment]
Hi @mzur, I do not experience this bug (neither on the _master_ nor on the currently developed _geo_upload_ branch). I'm using Firefox browser (v.129.0.2).

**Comment by mzur** [comment]
On biigle.de?

**Comment by mtiessen1175** [comment]
> On biigle.de?

No, I only had a look at the locally built instance, sorry for the confusion!

**Comment by mzur** [comment]
I see it on `master` but not on `geo_upload`. Did you run `npm install` and `npm run prod` on `master`?
